app/services/create_user_service.rb

```diff
@@ -1,23 +1,17 @@
 class CreateUserService
   def self.invite_responsible_body_user(user_params = {})
-    user = User.new(user_params.merge(override_params))
+    user = User.new(user_params)
```

Comment:
I don't think we don't want to be making this change to this method. "All responsible body users get a TechSource account" is a part of the domain logic that correctly lived in the service layer. There isn't any use case for expositing either `ordered_devices` or `approved_at` outside the service.

Reply:
I see, I've changed `invite_responsible_body_user ` back to the original